portfolio/src/main/java/com/google/sps/servlets/HelloWorldServlet.java

```diff
@@ -7,7 +7,7 @@
 import javax.servlet.http.HttpServletResponse;
 
 /** Handles requests sent to the /hello URL. Try running a server and navigating to /hello! */
-@WebServlet("/hello")
+@WebServlet("/facts.html")
```

Comment:
I think with the web servlet, this opens up the end point that is specified.
Generally there will be no `.html` in the endpoint because that specifies a webpage instead of an API endpoint

The endpoint you create here also sets up the url mapping so the` facts.html` file below shouldn't be needed

Reply:
Thanks so much! This fixed my servlet.